jetbrains-core/src/software/aws/toolkits/jetbrains/services/lambda/execution/local/LocalLambdaRunConfiguration.kt

```diff
@@ -77,25 +77,36 @@ class LocalLambdaRunConfiguration(project: Project, factory: ConfigurationFactor
     }
 
     override fun checkConfiguration() {
-        checkSamVersion()
         resolveRegion()
         resolveCredentials()
-        checkLambdaHandler()
+        val runtime = checkLambdaHandler()
```

Comment:
I understand this is where we currently get runtime, but it being returned by check is just weird, can we grab runtime in checkSamVersion as well?

Reply:
we should probably extract it then pass it in if we do that